### DISCUSSION TITLE
[Suggestion] Add a static analysis method in MSTG-CODE-2

### DISCUSSION THREAD

**Discussion by sohsatoh** [OWASP/mastg#2294] [Ideas]
Currently, the static analysis section of MSTG-CODE-2 for iOS only describes methods for those who can access to the source code.

However, there is a way to check that `get-task-allow` is true in the entitlement using ldid.

https://developer.apple.com/documentation/security/notarizing_macos_software_before_distribution/resolving_common_notarization_issues?language=objc

Any thoughts on adding this to the Static Analysis section?

(I wasn't sure whether I could submit a pull request directly, so I posted here.)

**Comment by cpholguera** [comment]
Hi @sohsatoh, you're absolutely right. Thanks for your suggestion, I've created an issue for it.
